cypress/e2e/backoffice/order-management/marketplace-payment-oms-flow-dms.cy.ts

```diff
@@ -0,0 +1,226 @@
+import { CatalogPage, ProductPage } from '@pages/yves';
+
+(Cypress.env('isDynamicStoreEnabled') ? describe : describe.skip)(
+  'marketplace payment OMS flow dms',
+  { tags: '@dms' },
+  () => {
+    (['b2c', 'b2b'].includes(Cypress.env('repositoryId')) ? describe.skip : describe)(
+      'marketplace payment OMS flow',
+      { tags: ['@backoffice'] },
+      (): void => {
+        const catalogPage = container.get(CatalogPage);
+        const productsPage = container.get(ProductPage);
+        const salesIndexPage = container.get(SalesIndexPage);
+        const salesDetailPage = container.get(SalesDetailPage);
+        const salesOrdersPage = container.get(SalesOrdersPage);
+        const userLoginScenario = container.get(UserLoginScenario);
+        const customerLoginScenario = container.get(CustomerLoginScenario);
+        const checkoutMpScenario = container.get(CheckoutMpScenario);
+        const merchantUserLoginScenario = container.get(MerchantUserLoginScenario);
+        const createStoreScenario = container.get(CreateStoreScenario);
+        const selectStoreScenario = container.get(SelectStoreScenario);
+        const enableCmsBlockForAllStoresScenario = container.get(EnableCmsBlockForAllStoresScenario);
+        const enableWarehouseForAllStoresScenario = container.get(EnableWarehouseForAllStoresScenario);
+        const enableProductForAllStoresScenario = container.get(EnableProductForAllStoresScenario);
+        const enableShipmentMethodForAllStoresScenario = container.get(EnableAllShipmentMethodsForAllStoresScenario);
+        const enablePaymentMethodForAllStoresScenario = container.get(EnableAllPaymentMethodsForAllStoresScenario);
+        const enableMerchantForAllStoresScenario = container.get(EnableMerchantForAllStoresScenario);
+        const enableShipmentTypeForAllStoresScenario = container.get(EnableShipmentTypeForAllStoresScenario);
+        const ensureCatalogVisibilityScenario = container.get(EnsureCatalogVisibilityScenario);
+
+        let staticFixtures: MarketplacePaymentOmsFlowStaticFixtures;
+        let dynamicFixtures: MarketplacePaymentOmsFlowDynamicFixtures;
+
+        before((): void => {
+          ({ staticFixtures, dynamicFixtures } = Cypress.env());
+
+          userLoginScenario.execute({
+            username: dynamicFixtures.rootUser.username,
+            password: staticFixtures.defaultPassword,
+          });
+
+          createStoreScenario.execute({
+            store: staticFixtures.store,
+          });
+
+          assignStoreRelationToExistingProduct();
+
+          selectStoreScenario.execute(staticFixtures.store.name);
+            ensureCatalogVisibilityScenario.execute();
+
+          if (!['b2c', 'b2b', 'b2b-mp'].includes(Cypress.env('repositoryId'))) {
+            enableShipmentTypeForAllStoresScenario.execute({
+              store: staticFixtures.store.name,
+              username: 'admin@spryker.com',
```

Comment:
Email is hardcoded. Previously dynamicFitures.rootUser.username was used, but auth was not successful. I tried to fix it and saw that it requires some time for me. Maybe you know how to do it fast, @dmiseev